draft-ietf-quic-qpack.md

```diff
@@ -1787,8 +1789,8 @@ for line in field_lines:
 
 # encode the prefix
 if requiredInsertCount == 0:
-  encodeIndexReference(prefixBuffer, 0, 0, 8)
-  encodeIndexReference(prefixBuffer, 0, 0, 7)
+  encodeInteger(prefixBuffer, 0, 0, 8)
+  encodeInteger(prefixBuffer, 0, 0, 7)
```

Comment:
encodeIndexReference() is shown earlier with a different signature.  I know pythonic things do have optional parameters, but this seems to match the encodeInteger() signature perfectly, so I assume it was just an incomplete edit.  Though, stylistically perhaps the second argument should be `0x00` rather than a literal `0`.

Reply:
This is a good catch, thanks.  The prefix is integers, not references.